en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md

```diff
@@ -24,132 +24,112 @@ doi: TBC
-
-Microsoft's Azure Cognitive Services can be harnessed to transcribe both typed text and handwriting as well as a combination of both. It can transcribe documents such as diaries, letters, forms, logbooks and research notes. I have used it to transcribe maps and diagrams as well. The potential uses for Digital History are numerous! Transcription with Azure Cognitive Services is well documented, but does require programming, hence this tutorial.
+Microsoft's Azure Cognitive Services can be harnessed to transcribe typed text, handwriting, or a combination of both. It can transcribe diaries, letters, forms, logbooks and research notes. I have also used it to transcribe maps and diagrams. The potential uses for Digital History are numerous! Transcription with Azure Cognitive Services is well documented, but does require some programming, hence this tutorial.
 
 ## Prerequisites
 
-+ Knowledge of Python is not required since all of the code is provided in the tutorial. That said, basic Python knowledge would be useful for users who wish to understand the code or to tweak it for their purposes.
-+ [Google Colab](https://colab.research.google.com/), a web-based virtual Python programming platform, was used to write this lesson. If you choose to use Google Colab to program Python (recommended), a Google account is required. If you choose to run the code in this tutorial locally on your own machine, Python 3.x and pip need to be installed.
++ Knowledge of Python is not required, since all of the code is provided in the tutorial. That said, basic Python knowledge would be useful for users who wish to understand the code or to tweak it for their purposes.
++ This lesson was written with [Google Colab](https://colab.research.google.com/), a web-based virtual Python programming platform. If you choose to use Google Colab to program Python (which I recommended), you will need a Google account. If you choose to run the code in this tutorial locally on your own machine, you will need to install [Python 3.x](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/).
 + An internet connection.
 + A credit card or debit card. (Though there is a free tier of service for Microsoft, you are required to put a credit or debit card on file. The card is not charged if the number of files processed is below 5,000 each month.)
 + A telephone number. (This is to verify your identity.)
 
-## Procedure
+## Setting up Azure Computer Vision
```

Comment:
In dialogue with Alex, I have split up “Procedure” into a “Setting Up” section and a “Projects” section. I thought this gave a clearer indication to the reader of what they could expect from each section. It also meant there were fewer embedded sections in the previous "6. Transcribe Handwriting" section, which allows us to avoid roman numerals.